sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/python/GpuArrowPythonRunner.scala

```diff
@@ -206,7 +129,7 @@ abstract class GpuPythonRunnerBase[IN](
 /**
  * Similar to `PythonUDFRunner`, but exchange data with Python worker via Arrow stream.
  */
-abstract class GpuArrowPythonRunnerBase(
+abstract class GpuArrowPythonRunnerBaseBase(
```

Comment:
Having a BaseBase class is a bit odd, especially since GpuArrowPythonRunnerBase is so trivially used.  I think this would be cleaner if GpuArrowPythonRunnerBase became GpuArrowPythonRunner and this can be GpuArrowPythonRunnerBase.

Reply:
Thank you for the suggestion 👍 
This was a placeholder though